content/pages/index.md

```diff
@@ -43,3 +43,13 @@ This package collects utilities that require both `xlrd` and `xlwt`, including t
 ## The Mailing List / Discussion Group
 
 There is a [Google Group](http://groups.google.com/group/python-excel) dedicated to working with Excel files in Python, including the libraries listed above along with manipulating the Excel application via COM. 
+
+## Commercial Development
+
+The following companies can provide commercial software development and consultancy and are specialists in working with Excel files in Python:
+
+<div class="affiliate-links"></div>
+
+| [![Clark Consulting & Research]({static}/images/ccr_python_excel.png)](http://www.clark-consulting.eu/) | [LingInfo](http://www.lexicon.net/sjmachin){.linginfo} |
+| ------------------------------------------------------------------------------------------------------- | --- |
```

Comment:
@sjmachin has retired, so no need for the lexicon.net link.

Reply:
ok, I'll take that off then

Reply:
done!